--- /etc/mkinitcpio.conf	2021-12-03 18:11:24.000000000 +0800
+++ mkinitcpio_patch_amdgpu.conf	2022-05-02 11:00:06.170638942 +0800
@@ -4,7 +4,7 @@
 # run.  Advanced users may wish to specify all system modules
 # in this array.  For instance:
 #     MODULES=(piix ide_disk reiserfs)
-MODULES=()
+MODULES=(amdgpu radeon)
 
 # BINARIES
 # This setting includes any additional binaries a given user may
@@ -49,7 +49,7 @@
 #
 ##   NOTE: If you have /usr on a separate partition, you MUST include the
 #    usr, fsck and shutdown hooks.
-HOOKS=(base udev autodetect modconf block filesystems keyboard fsck)
+HOOKS=(base udev resume autodetect modconf block filesystems keyboard fsck)
 
 # COMPRESSION
 # Use this to compress the initramfs image. By default, zstd compression
